### DISCUSSION TITLE
Is this possible to get onClick in each row?

### DISCUSSION THREAD

**Discussion by italosantana** [KevinVandy/material-react-table#62] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/48260257/188157267-0b01a0b0-bead-426e-9b73-f375958188b3.png)

For each line I want to put a 'clickable' link to each profile.

It is possible?

**Comment by KevinVandy** [comment, marked as answer]
Of course, just use a custom `Cell` render. You can see where I did exactly that for my [props table on the docs site](https://www.material-react-table.com/docs/api/state-options). Scroll down to see the full source code on that page, but here is the snippet you want.

```jsx
{
          accessorKey: 'link',
          header: 'More Info Links',
          Cell: ({ cell, row }) => (
            <Link href={cell.getValue<string>()}>
                {row.original?.linkText}
            </Link>
          ),
        },
```

**Reply by KevinVandy** [reply]
Also be sure to read the full table even listeners docs to learn how to add an `onClick` to any table component https://www.material-react-table.com/docs/guides/table-event-listeners
